packages/documentation/package.json

```diff
@@ -59,7 +58,6 @@
   ],
   "license": "MIT",
   "scripts": {
-    "postinstall": "cd node_modules/web-components && yarn install --production && yarn build",
```

Comment:
No longer needed with the updated `web-components` version; we moved this to that library's `prepare` script where it belongs.